kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -1,30 +1,357 @@
 <template>
 
   <div>
-    <p>enroll for class {{ this.classId}}</p>
+    <div class="top-buttons pure-g">
+
+      <div :class="windowSize.breakpoint > 2 ? 'pure-u-1-2' : 'pure-u-1-1 align-center'">
+        <router-link :to="editClassLink">
+          <icon-button
+            :text="$tr('backToClassDetails')"
+            :primary="false">
+            <mat-svg category="navigation" name="arrow_back"/>
+          </icon-button>
```

Comment:
styling of this button is a little off:

![image](https://cloud.githubusercontent.com/assets/2367265/23730817/80041d76-041e-11e7-8a6c-6932ef2eec0c.png)